src/pages/videoListing/VideoListing.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react'
+import Navbar from 'components/navbar/Navbar'
+import Footer from 'components/footer/Footer'
+import VideoCard from 'components/videocard/VideoCard'
+
+export default function VideoListing() {
+  const [videos, setVideos] = useState([])
+  useEffect(() => {
+    ;(async () => {
+      const response = await fetch('/api/videos')
+      const data = await response.json()
+      setVideos(data.videos)
+    })()
+  })
```

Comment:
dependency array missing in useeffect. Please check once do you need it or not. Otherwise it will run continuously.

Reply:
Prettier is putting it on save, unable to remove it